app/models/models.py

```diff
@@ -35,7 +39,7 @@ class Voter(db.Base):
 
 class DuplicatedVoter(db.Base):
     __tablename__ = "duplicated_voters"
-    pass
+    id = Column(String(50), primary_key=True, index=True, default=uuid4().hex)
```

Comment:
Like earlier mentioned lets have a constant `UUID_HEX=uuid4().hex` defined at the top of the module that we can reuse, lets keep it DRY